Doc/library/compileall.rst

```diff
@@ -83,6 +83,15 @@ compile Python sources.
    If ``0`` is used, then the result of :func:`os.cpu_count()`
    will be used.
 
+.. cmdoption:: --invalidation-mode [timestamp|checked-hash|unchecked-hash]
+
+   Control how the generated pycs will be invalidated at runtime. The default
+   setting, ``timestamp``, means that pyc files with the source timestamp and
```

Comment:
I just wanted to double-check that we are referring to the files as "pyc" and "pycs" throughout the rest of the docs instead of "`.pyc` files" or something? Basically just checking we're being consistent.